app/javascript/components/userinput.vue

```diff
@@ -0,0 +1,117 @@
+<template lang="html">
+  <div class="user-input position-relative">
+    <input type="text" class="d-none" :name="name" :value="selectedSuggestion.id">
+    <input type="text" class="form-control" v-model="query" placeholder="Begin met typen..."
+           @input="updateValue"
+           @keydown.enter="selectFirstSuggestion"
+           aria-haspopup="true" v-bind:aria-expanded="dropdownOpened"
+           required="true" autocomplete="off">
+
+    <div class="row">
+      <ul class="dropdown-menu" v-bind:class="{'show':dropdownOpened}">
+        <li class="dropdown-item" v-for="(suggestion, index) in suggestions"
+            v-on:click.prevent="suggestionClicked(index)">
+          <a href="#">{{ suggestion.name }}</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    props: {
+      name: {
+        type: String
+      },
+      value: {
+        type: Object,
+        default: {
+          id: 0
+        }
+      },
+      includePayment: {
```

Comment:
Might be better to call it something like `inlucudeNonUserPayment` (maybe bit long) or `includePin` and `includeCash`, with users is it also payment, so payment is a bit vague

Reply:
Added includePin/includeCash